src/topic.ts

```diff
@@ -592,13 +597,27 @@ export class Topic {
     );
   }
 
-  publish(data: Buffer, attributes?: Attributes): Promise<string>;
+  publish(data: Buffer, options?: PubsubMessage): Promise<[string]>;
   publish(data: Buffer, callback: PublishCallback): void;
   publish(
     data: Buffer,
-    attributes: Attributes,
+    options: PubsubMessage,
```

Comment:
Why is the options parameter of type PubsubMessage? Is it supposed to be MessageOptions?

Reply:
`PubsubMessage` is just the name of the generated type we get from the protos. I can update it though since `MessageOptions` definitely sounds concise.